hassio/src/addon-view/hassio-addon-audio.ts

```diff
@@ -0,0 +1,207 @@
+import "web-animations-js/web-animations-next-lite.min";
+
+import "@material/mwc-button";
+import "@polymer/paper-card/paper-card";
+import "@polymer/paper-dropdown-menu/paper-dropdown-menu";
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-listbox/paper-listbox";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+} from "lit-element";
+
+import { HomeAssistant } from "../../../src/types";
+import {
+  HassioAddonDetails,
+  HassioHardwareAudioDevice,
+  fetchHassioHardwareAudio,
+  setHassioAddonOption,
+  HassioAddonSetOptionParams,
+} from "../../../src/data/hassio";
+import { PolymerChangedEvent } from "../../../src/polymer-types";
+import { hassioStyle } from "../resources/hassio-style";
+import { haStyle } from "../../../src/resources/styles";
+
+@customElement("hassio-addon-audio")
+class HassioAddonAudio extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public addon!: HassioAddonDetails;
+  @property() protected error?: string;
+  @property() private _inputDevices?: HassioHardwareAudioDevice[];
+  @property() private _outputDevices?: HassioHardwareAudioDevice[];
+  @property() private _selectedInput?: string;
+  @property() private _selectedOutput?: string;
+
+  protected render(): TemplateResult | void {
+    return html`
+      <paper-card heading="Audio">
+        <div class="card-content">
+          ${this.error
+            ? html`
+                <div class="errors">${this.error}</div>
+              `
+            : ""}
+
+          <paper-dropdown-menu
+            label="Input"
+            @value-changed="${this._setInputDevice}"
+          >
+            <paper-listbox
+              slot="dropdown-content"
+              attr-for-selected="device"
```

Comment:
Drop this as you already set `.selected`

Reply:
If I drop that, it does not display the selected device on load, did a search through the codebase and it looks like all other places have both `.selected` and `attr-for-selected`